plugin/storage/es/mappings/mapping.go

```diff
@@ -31,23 +32,58 @@ type MappingBuilder struct {
 	ILMPolicyName                string
 }
 
+// MappingType represents the type of Elasticsearch mapping
+type MappingType int
+
+const (
+	SpanMapping MappingType = iota
+	ServiceMapping
+	DependenciesMapping
+	SamplingMapping
+)
+
+func (mt MappingType) String() (string, error) {
+	switch mt {
+	case SpanMapping:
+		return "jaeger-span", nil
+	case ServiceMapping:
+		return "jaeger-service", nil
+	case DependenciesMapping:
+		return "jaeger-dependencies", nil
+	case SamplingMapping:
+		return "jaeger-sampling", nil
+	default:
+		return "", fmt.Errorf("Unknown mapping type %d", mt)
```

Comment:
String() methods usually do not return errors. You can return `"unknown"` string.